blog/Automating tasks with Python.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Automating Tasks with Python: Using the OS and Subprocess Modules'
+sidebar_label: Automating Tasks with Python
+authors: [nayanika-mukherjee   ]
```

Comment:
Error

```bash
[cause]: Error: Blog author with key "nayanika-mukherjee" not found in the authors map file.
#14 4.015     Valid author keys are:
#14 4.015     - ajay-dhangar
#14 4.015     - ajay-dhangar_2024
#14 4.015     - ajay-dhangar_2020
#14 4.015     - hitesh-gahanolia
#14 4.015     - dharshibalasubramaniyam
#14 4.015     - abhijith-m-s
#14 4.015     - khushi-kalra
#14 4.015     - nayanikamukherjee
#14 4.015     - pujan-sarkar
#14 4.015     - mohitmuktikant
#14 4.015     - santhosh-siddhardha
```
 
So, you need define a author it's name is `nayanika-mukherjee`. 

add your info in author file [authors.yml](https://github.com/CodeHarborHub/codeharborhub.github.io/blob/main/blog/authors.yml)